### PR TITLE
Update GPT Integration

### PR DESCRIPTION
**Pull request by kevinfodness**
* Use Google's recommendation for including the script statically
* Remove `enableAsyncRendering` call, as the current version of GPT is async by default and this function does nothing
* Add a filter for whether `enableServices` should be called in the head or not

### CODE REVIEW

File: inc/ad-servers/class-ad-layers-dfp.php [hunk 1]
```diff
@@ -282,9 +267,11 @@ public function header_setup() {
 				do_action( 'ad_layers_dfp_custom_targeting' );
 				?>
 
-				if ( typeof AdLayersAPI === 'undefined' || ! AdLayersAPI.isDebug() ) {
-					googletag.enableServices();
-				}
+				<?php if ( apply_filters( 'ad_layers_dfp_enable_services_in_head', true, $this ) ) : ?>
```
Comment (mboynes) [inline inc/ad-servers/class-ad-layers-dfp.php:270]: I know none of the other filters are documented, but there's no better opportunity than with this one to add a docblock for the filter 😄

**Review by mboynes — APPROVED** [review]
:+1:

File: inc/ad-servers/class-ad-layers-dfp.php (hunk 1 above)
Comment (kevinfodness) [reply inc/ad-servers/class-ad-layers-dfp.php:270]: Good call!

**Review by montchr — APPROVED** [review]
🚀

**Pull request merged**